dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -70,6 +70,14 @@ def handle_pending_messages
     AsyncProgressHandler.stubs(:progress_queue).returns(@fake_queue)
   end
 
+  # TODO(asher): This probably should be refactored to a more reusable location within our dashboard
+  # testing code.
+  # @return [Integer, nil] The ID of the user signed in (per session warden), or nil if no user is
+  #   signed in.
+  def current_user_id
+    session['warden.user.user.key'].try(:first).try(:first)
+  end
```

Comment:
Because I'd rather keep the PR changes self-contained. As part of moving it to `test_helper`, I'd want to convert other tests to make use of it. Another PR.